packages/xdl/src/Project.js

```diff
@@ -1892,6 +1894,38 @@ async function stopWebpackServerAsync(projectRoot) {
   });
 }
 
+export async function bundleWebpackAsync(projectRoot, packagerOpts) {
+  let { exp } = await ProjectUtils.readConfigJsonAsync(projectRoot);
+  if (!exp.platforms.includes('web')) {
+    return;
+  }
+  let config = webpackConfig({
+    projectRoot,
+    development: packagerOpts.dev,
+    production: !packagerOpts.dev,
+  });
+  let compiler = webpack(config);
+
+  process.env.BABEL_ENV = 'production';
+  process.env.NODE_ENV = 'production';
```

Comment:
I guess these should also be set according to the `packagerOpts.dev` option like the webpack config above?